File: 1des_old/fpoo/workspace/CorrecaoLista1e2/src/lista1/Ex2.java
package lista1;

import java.util.Scanner;

/*Desenvolva um programa que leia a
 * velocidade de um carro (km/h) e a
 * distância(Km) a ser per-corrida (km)
 * por ele. Calcule e apresente na tela,
 * quanto tempo (horas) será necessário
 * para o carro percorrer a distância
 * informada*/
public class Ex2 {
	public static void main(String[] args) {
		Scanner info = new Scanner(System.in);
		
		System.out.print("Velocidade Km/h: ");
		int velocidade = info.nextInt();
		System.out.print("Distância Km: ");
		int distancia = info.nextInt();
		
		int tempo = distancia / velocidade;
		
		System.out.print("O tempo gasto será de: "+tempo+" horas");
	}
}

File: 1des/fpoo/aula03/ex2.c
/*
Desenvolva um programa que leia a velocidade de um carro (km/h)
e a distância a ser percorrida (km) por ele.
Calcule e apresente na tela, quanto tempo (horas) será necessário
para o carro percorrer a distância informada.
*/

#include <stdio.h>
#include <locale.h>

int main(){
	setlocale(LC_ALL,"");
	int velocidade, distancia;
	float tempo;
	//Entradas
	printf("Digite a velocidade do carro em km/h: ");
	scanf("%d",&velocidade);
	printf("Digite a distância em km: ");
	scanf("%d",&distancia);
	//Processamento
	tempo = (float) distancia / velocidade;
	//Saída
	printf("Esta distância será percorrida em %.1f horas",tempo);
}
